gaas-ui/test/rest/clients/auth-client-factory.test.ts

```diff
@@ -1,26 +1,26 @@
-import { AuthApiClient } from "../../../src/rest/clients/auth-api-client";
-import { AuthClientFactory } from "../../../src/rest/clients/auth-client-factory";
-import { IAuthClient } from "../../../src/rest/clients/authclient";
-import { CognitoClient } from "../../../src/rest/clients/cognito-client";
-import { Config } from "../../../src/rest/config";
+import { AuthApiClient } from '../../../src/rest/clients/auth-api-client';
+import { AuthClientFactory } from '../../../src/rest/clients/auth-client-factory';
```

Comment:
Why is this test single quotes?

Reply:
Shouldn't it have been caught?

Reply:
Hmm yeah interesting. Maybe .ts files aren't being picked up correctly.

Reply:
So test files were not being auto corrected by the linter. Have updated now so everything should be formatted correctly. Sorry this does mean more files being added just with formatting changes.